src/bikeracks/__init__.py

```diff
@@ -86,21 +86,21 @@ def submitFeedback():
     @app.route('/submitRemovalSuggestion', methods=('POST',))
     def submitSuggestion():
         rack_id = request.form.get("rack_id", type=float)
-        removal_reason = request.form.get("removal_reason", type=int)
+        reason_id = request.form.get("reason_id", type=int)
         user_id = request.form.get("user_id", type=str)
 
         # connect to database
         db = get_db()
-        
+
         try:
-        #rack_id, user_id, removal_reason, time_stamp
+        #rack_id, user_id, reason_id, time_stamp
```

Comment:
Why have this comment here? The query is right below it and authoritative.

Reply:
Removed.